packages/eslint-config-react/typescript.js

```diff
@@ -38,10 +39,12 @@ module.exports = {
         },
       },
     ],
+    '@typescript-eslint/no-explicit-any': 'warn',
```

Comment:
This should be kept as an error

Reply:
We can change it later but it's breaking change. it's was in warn before v6